keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/ResourceMetadata.kt

```diff
@@ -40,3 +42,7 @@ data class ResourceMetadata(
     ) + data)
       .toString()
 }
+
+data class SubmittedMetadata(
+  val serviceAccount: String
```

Comment:
I feel like this is one case where a good old `Map<String, Any>` might actually be appropriate

Reply:
It's easier to do auth checking when this is an actual class with actual values. I am having trouble getting it to work with a map, so I'm just gonna leave it as is for now.